ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/DeploymentsTable.tsx

```diff
@@ -138,7 +187,7 @@ function DeploymentsTable({
                                         </Link>
                                     </Td>
                                     {showCveDetailFields && (
-                                        <Td>
+                                        <Td className={getVisibilityClass('cvesBySeverity')}>
```

Comment:
Thinking about naming compared to other names related to table cells caused me to notice absence of `dataLabel` prop, except for **Deployment** column (if you expand unchanged code above).
![deployments_dataLabel](https://github.com/user-attachments/assets/413bc444-b8a0-4764-a6b7-f97d6f7af7e9)

Reply:
If you decide to add `dataLabel` prop while you are editing `Td` elements, also ImagesTable.tsx file.

Reply:
Good call, will do.

(Good candidate for a lint rule?)

Reply:
Indeed, wrote first draft of `Td-dataLabel` which finds 51 errors in 19 files.